install/kubernetes/helm/istio-remote/requirements.yaml

```diff
@@ -7,3 +7,7 @@ dependencies:
     version: 1.1.0
     condition: security.enabled
     repository: file://../subcharts/security
+  - name: istio-cni
+    version: ">=0.0.1"
+    repository: https://raw.githubusercontent.com/istio/istio.io/master/static/charts
```

Comment:
Wouldn't be simpler if we kept all istio helm settings in the same repo ? It doesn't have to be istio/istio...